src/libraries/Native/Unix/System.Native/pal_threading.h

```diff
@@ -0,0 +1,133 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma once
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include <pthread.h>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// LowLevelMutex - Represents a non-recursive mutex
+
+typedef struct
+{
+    pthread_mutex_t m_mutex;
+
+#ifdef DEBUG
+    bool m_isLocked;
+#endif
+} LowLevelMutex;
+
+bool LowLevelMutex_TryInitialize(LowLevelMutex *mutex);
+bool LowLevelMutex_TryInitialize(LowLevelMutex *mutex)
```

Comment:
C does not have the One Definition Rule.  All these functions in the header should be `static inline`

Reply:
I see, thanks that worked